content/powershell/concepts/loops/loops.md

```diff
@@ -1,13 +1,131 @@
 ---
 Title: 'Loops'
-Description: ''
+Description: 'Loops (for, foreach, while, do-while, do-until) execute the same statements multiple times until a stop condition is met.'
 Subjects:
+  - 'Code Foundations'
   - 'Computer Science'
+  - 'Bash/Shell'
 Tags:
+  - 'Break'
+  - 'Continue'
+  - 'For'
+  - 'ForEach'
   - 'Loops'
   - 'While'
-  - 'For'
 CatalogContent:
   - 'learn-powershell'
   - 'paths/computer-science'
 ---
+
+**Loops** enable programs to execute the same set of statements multiple times until a stop condition is met. There are five types of loops in PowerShell:
+
+- `for`
+- `while`
+- `do`..`while`
+- `do`..`until`
+- `foreach`
+
+## `for` Loop
```

Comment:
same note as above

Reply:
Fixed in 23f54309efceaf758722ab60f2a5dde33e0f9e65